scripts/utils/migrate_includes.py

```diff
@@ -15,9 +15,10 @@
 from pathlib import Path
 import re
 import sys
+import os
 
 
-ZEPHYR_BASE = Path(__file__).parents[2]
```

Comment:
Since Python 3.4 `__file__` is an absolute path, ref. https://docs.python.org/3.4/whatsnew/3.4.html#other-language-changes